docs/UserGuide.md

```diff
@@ -121,6 +133,7 @@ Format: `note INDEX note/NOTE`
 Examples:
 *  `note 1 note/History of asthma` - Edits the note of the 1st client to `History of asthma`.
 *  `edit 2 note/Previously sprained both ankles` - Edits the note of the 2nd client to `Previously sprained both ankles`.
```

Comment:
I know you didn't change this line, but should we change `edit` to `note` here since it's in the note section?
```suggestion
*  `note 2 note/Previously sprained both ankles` - Edits the note of the 2nd client to `Previously sprained both ankles`.
```

Reply:
committed your change, thank you @dillontkh